pointpats/spacetime.py

```diff
@@ -5,14 +5,20 @@
 __author__ = "Nicholas Malizia <nmalizia@asu.edu>", "Sergio J. Rey \
 <srey@asu.edu>", "Philip Stephens <philip.stephens@asu.edu"
 
-__all__ = ['SpaceTimeEvents', 'knox', 'mantel', 'jacquez', 'modified_knox']
+__all__ = ['SpaceTimeEvents', 'knox', 'mantel', 'jacquez', 'modified_knox',
+           'Knox', 'KnoxLocal]
```

Comment:
```suggestion
           'Knox', 'KnoxLocal']
```
this causes import error now